Library/Homebrew/test/utils/shell_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "utils/shell"
+
+describe Utils::Shell do
+  describe "::path_to_shell" do
```

Comment:
Huh, okay, leave it as-is then.

I think I picked up my style from [here](https://github.com/thoughtbot/guides/tree/master/style/testing#unit-tests), way back in the day.